deploy/gke/redaction/README.md

```diff
@@ -68,4 +68,105 @@ When you are done, you can clean up everything you've done with the following st
 
 ```
 kubectl delete namespace $OTEL_NAMESPACE
-```
\ No newline at end of file
+```
+
+## Try it out yourself (Optional)
+
+This section contains steps that you can follow to actually see redaction you setup earlier in action. Since there is no expected outcome from our kubernetes deployment due to a lack of telemetry data, the redaction processor setup in our collector cofiguration is not applied to anything. In this section we will introduce a source of telemetry data to actually see the redaction in action.  
+
+### Prerequisites 
+You should be able to follow all steps mentioned in this README till [Verify the Deployment](#verify-the-deployment) successfully. This ensures that there are no permissions issues and you are able to succeesfully run a GKE cluster and connect to it. 
+
+### Source of telemetry data
+
+We would be using JSON files containing telemetry data in OTLP format as the source of telemetry data. A sample file has been provided here - [testdata.json](./otlp-data/testdata.json). The collector running in the cluster would read this file and treat the data in this file as if it were coming from a running application. 
```

Comment:
```suggestion
We will be using JSON files containing telemetry data in OTLP format as the source of telemetry data. A sample file has been provided here - [testdata.json](./otlp-data/testdata.json). The collector running in the cluster will read this file and treat the data in this file as if it is coming from a running application. 
```

It sounds a little clearer at this point to assume the reader is actually following the guide (ie, "you will do this step" vs "you would do this step")